test/services/licences/supplementary/process-billing-flag.service.test.js

```diff
@@ -275,6 +276,86 @@ describe('Process Billing Flag Service', () => {
         })
       })
     })
+
+    describe('with a licence Id', () => {
+      before(() => {
+        payload = {
+          licenceId: 'b5f81330-bec5-4c3e-95dd-267c10836fea'
+        }
+      })
+
+      describe('that should be flagged for two-part tariff supplementary billing', () => {
```

Comment:
I'm not really getting this test. The `DetermineLicenceFlagsService` always sets `flagForTwoPartTariffSupplementary` to false and never returns a start date. So no years could be calculated for 2PT?

Reply:
Another good spot, my bad! Thank you